spec/features/parks/02_parks_show_spec.rb

```diff
@@ -0,0 +1,41 @@
+require "rails_helper"
+#
+# As an authenticated user
+# I want to view a list of items
+# So that I can pick items to review
+
+feature "authenticated users can view parks" do
+  scenario "visitor signs in and views parks on index page" do
+    parks = FactoryGirl.create_list(:park, 3)
+
+    visit root_path
+
+    expect(page).to have_content "Parks"
+
+    parks.each do |park|
+      visit park_path(park)
+      expect(page).to have_content(park.title)
+      expect(page).to have_content(park.location)
+      expect(page).to have_content(park.rating)
+    end
+  end
+
+  let(:park){ Park.create!(title: "Commons", location: "Boston", rating: 10, bathroom: true, picnic: true, pets: true, basketball: true, baseball: true)}
```

Comment:
Line is too long. [153/80]<br>Space missing to the left of {.<br>Space missing inside }.